src/components/notifications/AppSelector/AppSelector.scss

```diff
@@ -171,6 +170,53 @@
       }
     }
 
+    &-item-skeleton {
```

Comment:
I should improve these, seems like we don't need all

Reply:
Seems like we don't all what?

Reply:
No worries, made refactors, all good now I think